Bynder/Test/Bynder.Test.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsTestProject>true</IsTestProject>
-    <AssemblyVersion>2.0.0</AssemblyVersion>
-    <FileVersion>2.0.0</FileVersion>
+    <AssemblyVersion>3.1.100</AssemblyVersion>
+    <FileVersion>3.1.100</FileVersion>
```

Comment:
Aren't `AssemblyVersion` and `FileVersion` about versions of the project itself? As opposed to the .NET version? In Visual Studio I can't see them in any way, or if they are even used for something..

Reply:
My understanding was that the assembly version represented the assemblies in th dotnet build. However if I remove them it makes no difference while building the tests. 

So, I still do not understand the extent of its usefulness tbh. I will get rid of it since it affects nothing.